pkg/api/plugin.go

```diff
@@ -32,7 +32,7 @@ type Plugin interface {
 
 	// GenerateConfig ensures all the necessary in-cluster config is generated
 	// for an Openshift cluster.
-	GenerateConfig(ctx context.Context, cs *OpenShiftManagedCluster) error
+	GenerateConfig(ctx context.Context, cs, oldCS *OpenShiftManagedCluster) error
```

Comment:
doing this is unfortunately out of the question.  oldCS is not available on the RP backend.  You don't need it anyway.

Reply:
See last comment